components/formats-bsd/src/loci/formats/AxisGuesser.java

```diff
@@ -71,18 +73,24 @@
   public static final int S_AXIS = 4;
 
   /** Prefixes indicating space dimension. */
-  public static final String[] Z_PREFIXES = {
+  public static final ImmutableSet<String> Z_PREFIXES = ImmutableSet.of(
```

Comment:
If there's no reference to the mutable collection, then an unmodifiable view is fine.